ios/ReactAirplay/Modules/RAAirplayConnectivityContext.swift

```diff
@@ -28,22 +28,35 @@ class RAAirplayConnectivityContext: RAEventEmitter {
     return false
   }
 
-  @objc(fetchAirplayConnectivityWithResolver:rejecter:)
-  func fetchAirplayConnectivity(
+  @objc(fetchAirplayRoutesWithResolver:rejecter:)
+  func fetchAirplayRoutes(
```

Comment:
Can this method be named `fetchAVAudioSessionRoutes` since that's more correct

Reply:
Done.